js/models/payment-request.js

```diff
@@ -0,0 +1,35 @@
+var app = app || {};
+
+app.models = app.models || {};
+
+app.models.PaymentRequest = (function() {
+
+	'use strict';
+
+	return Backbone.Model.extend({
+        idAttribute: 'id',
+
+        defaults: {
+            currency: '',
+            address: '',
+            confirmed: '',
+            amount: '',
+            timestamp: (new Date).getTime()
+        },
+
+        localStorageKey: 'payment-requests',
+
+        save: function() {
+
+   			var data = this.toJSON();
+            var currentPaymentRequestsArr = JSON.parse(localStorage.getItem(this.localStorageKey));
+
+            if (currentPaymentRequestsArr && currentPaymentRequestsArr.length > 0) {
+                currentPaymentRequestsArr.push(data);
+                localStorage.setItem(this.localStorageKey, JSON.stringify(currentPaymentRequestsArr));
+            } else {
+                localStorage.setItem(this.localStorageKey, JSON.stringify([data]));
+            }
```

Comment:
I think all the local storage stuff can be moved to the payment requests collection instead. And it might be safer (for data integrity) to save each payment request as a separate local storage entry.